stories/birthDate.story.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
+import BirthDate from "../src/BirthDate";
+
+storiesOf("BirthDate", module).add(
+  "BirthDate",
+  withInfo("Simply shows the birthdate in form DD.MMM.YYYY")(() => (
```

Comment:
Oh wait small one, should this be DD.MM.YYYY? Or is it in the format Aug, Dec ?

Reply:
We use this format in 3 places as far as I know. I dont know what is nicer having two separate components with different format or one and we parse a prop on it and regarding this prop we show different format? 
Dont have a strong opinion here

Reply:
if you see the comments Madison asked for it

Reply:
Cool cool just curious